ios/autotools-ios-helper.sh

```diff
@@ -24,7 +24,7 @@ OQS_BUILD=${OQS_BUILD:-"$(pwd)/../liboqs/build_universal"}
 
 build() {
     # Compiler options
-    export OPT_FLAGS="-O3 -fno-inline"
+    export OPT_FLAGS="-O3"
```

Comment:
Might be worth mentioning in the commit message that this because unnecessary due to the second patch?